### PR TITLE
BOSE landing: first commit

### PR DESCRIPTION
**Pull request by NVyrsta**
[DEMO LINK](https://NVyrsta.github.io/layout_miami/)

### CODE REVIEW

**Comment by andrieieva-yuliia** [comment]
Your demo link doesn't work 😢

**Comment by NVyrsta** [comment]
> Your demo link doesn't work 😢

thanks, I fixed it :)

File: src/styles/blocks/contact.scss [hunk 1]
```diff
@@ -0,0 +1,118 @@
+.contact {
```
Comment (andrieieva-yuliia) [inline src/styles/blocks/contact.scss:1]: Screen Shot 2022-12-21 at 8 14 18 PM
The form should stretch across the width and the contact text should be aligned to the left side

File: src/styles/blocks/footer.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.footer {
```
Comment (andrieieva-yuliia) [inline src/styles/blocks/footer.scss:1]: Screen Shot 2022-12-21 at 7 48 47 PM
The same problem as with the header

**Review by andrieieva-yuliia — CHANGES_REQUESTED** [review]
In general, the work is excellent, small changes will make it even better  💣 👍

File: src/styles/blocks/header.scss [hunk 3]
```diff
@@ -0,0 +1,96 @@
+.header {
```
[the remark this answers is not in the document]
Comment (NVyrsta) [reply src/styles/blocks/header.scss:1]: Thank you!
Especially by npm
I would never find it there :) 😁

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]
Good job!
Some comments:
1) Use svg for icons
image
2) Inputs in form should be cleared after form submitting
3) Check `tab` button on the page with opened menu, I can go to the other elements via this button and they are visible, fix it
4) I see other images while opening mobile menu
image
5) Check redundant indents for the footer
image

**Review by Anna-Tkachenko — APPROVED** [review]
Well Done! ⭐️
Just one moment:
1) Check indents between image and text in `Recommended` section
image

**Pull request closed without merging**